docs/application/native/tutorials/details/tizen-apis.md

```diff
@@ -195,20 +195,20 @@ different API modules to decide which APIs to use.
 | [Application  Framework](../../../../org.tizen.native.wearable.apireference/group__CAPI__APPLICATION__FRAMEWORK.html) | [Application  Manager](../../../../org.tizen.native.wearable.apireference/group__CAPI__APPLICATION__MANAGER__MODULE.html) | Gets  information, such as the application ID and path to the shared data  directory, of the current application and other installed or running  applications. | The  Application Framework API contains submodule APIs for application  development. The submodule APIs enable application life-cycle management,  usage of functionality that is exported by other applications, and access to  a user's application preferences.<br>Guides:<br> - [Alarms and Scheduling](../../guides/alarm/alarm-cover.md)<br> - [Application Management](../../guides/app-management/app-management-cover.md)<br> - [Notifications and Content Sharing](../../guides/notification/notification-cover.md)    |
-| [Application  Framework](../../../../org.tizen.native.wearable.apireference/group__CAPI__APPLICATION__FRAMEWORK.html) | [Badge](../../../../org.tizen.native.wearable.apireference/group__BADGE__MODULE.html) | [Creates  and removes badges on the application's home screen icon. A badge  is an image displayed on the application icon, which informs the user about  notifications and events. This submodule can also be used to set and get the  badge value and visibility.](../../guides/app-management/app-icons.md) | The  Application Framework API contains submodule APIs for application  development. The submodule APIs enable application life-cycle management,  usage of functionality that is exported by other applications, and access to  a user's application preferences. <br>Guides:<br> - [Alarms and Scheduling](../../guides/alarm/alarm-cover.md)<br> - [Application Management](../../guides/app-management/app-management-cover.md)<br> - [Notifications and Content Sharing](../../guides/notification/notification-cover.md)   |
+| [Application  Framework](../../../../org.tizen.native.wearable.apireference/group__CAPI__APPLICATION__FRAMEWORK.html) | [Badge](../../../../org.tizen.native.wearable.apireference/group__BADGE__MODULE.html) | Creates  and removes [badges](../../guides/app-management/app-icons.md) on the application's home screen icon. A badge  is an image displayed on the application icon, which informs the user about  notifications and events. This submodule can also be used to set and get the  badge value and visibility. | The  Application Framework API contains submodule APIs for application  development. The submodule APIs enable application life-cycle management,  usage of functionality that is exported by other applications, and access to  a user's application preferences. <br>Guides:<br> - [Alarms and Scheduling](../../guides/alarm/alarm-cover.md)<br> - [Application Management](../../guides/app-management/app-management-cover.md)<br> - [Notifications and Content Sharing](../../guides/notification/notification-cover.md)   |
```

Comment:
alarm-cover.md should be changed into overview.md.
app-management-cover.md should be changed into index.md.
notification-cover.md should be changed into overview.md.

Reply:
All contents were moved to native/guides/index.md so I delete it.